src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,71 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
+    private static final String WHITESPACE = " ";
+    private static final String COMA = ",";
+    private static final int ZERO = 0;
```

Comment:
Please use appropriate naming for constants that represents meaning of value but not value itself:
```suggestion
    private static final int OPERATION_TYPE_INDEX = 0;
```
Please recheck all usages of ZERO, ONE, TWO. You may require different constants for same value if this value is used for different purposes

Reply:
Change

Reply:
not fixed